assets/css/event/animation.css

```diff
@@ -0,0 +1,36 @@
+@keyframes heartfade {
+  0% {
+    opacity: 1;
+  }
+
+  50% {
+    opacity: 0;
+  }
+}
+
+.heart {
+  z-index: 999;
+  animation: heartfade 6s linear;
+  position: absolute;
+}
+
+.heart:before {
+  content: "";
+  background-color: #fc2a62;
```

Comment:
Maybe it would be nice to use `--dark-font-color` to match the application UI (or maybe we don't want to merge this branch to `master` but to some demo branch, then it's wahtever)